src/main/kotlin/blackjack/Cards.kt

```diff
@@ -0,0 +1,44 @@
+package blackjack
+
+data class Cards(private val _cards: MutableList<Card> = mutableListOf()) {
+    val cards: List<Card>
+        get() = _cards.toList()
+
+    fun add(card: Card) {
+        _cards.add(card)
+    }
+
+    fun addAll(cards: List<Card>) {
+        _cards.addAll(cards)
+    }
+
+    fun bestScore(): Int {
+        return possibleScoreSum().filter { it <= BLACKJACK }.maxOrNull() ?: ZERO
+    }
```

Comment:
```suggestion
    fun bestScore(): Int {
        return possibleScoreSum().filter { it <= BLACKJACK }.max()
    }
```

max() / maxOrNull() 중에 maxOrNull를 사용한 이유가 있으실까요?

Reply:
BLACKJACK = 21로 지정을 했기 때문에 만약, 22점을 초과하는 스코어라면 max()에 의해 아무런 엘리먼트를 찾을 수 없어서 NoSuchElementException이 발생했기 때문이었습니다.

다만, 이건 제가 블랙잭에 대한 규칙을 잘 숙제하지 못한 탓에 이러한 문제가 발생됐다고 생각되는데요,
아래 코멘트에서 규칙과 관련하여 이야기 해주셨듯이 다음 PR에서 해당 내용을 개선해보겠습니다 ! 

> 블랙잭 게임은 카드의 합이 21이라고, 모두 블랙잭은 아니에요!
한번, 블랙잭에 대한 상태가 어떻게 만들어지는지 알아보시고, 적용해 보시겠어요?